developer-docs-site/docs/concepts/delegated_staking.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Delegated Staking"
+slug: " Delegated staking"
+---
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Delegated Staking
+
+## Delegated Staking on the Aptos Blockchain
+
+:::tip Consensus We strongly recommend that you read about [Staking](../concepts/staking.md) on the Aptos Blockchain first. :::
+
+Delegated staking is an extension of the staking protocol on Aptos. A delegation pool abstracts the stake owner to an entity capable of collecting stake from delegators and adding it on their behalf to the native stake pool attached to the validator. This means that multiple entities can stake to a stake pool to reach the minimum requirement for the validator to join the validator set. Delegators can add stake to an inactive pool, but the delegation pool will not earn rewards until it is active. 
+
+:::danger Delegation pools are permissionless and anyone can stake to the validator. You cannot change the type of stake pool once it's created. For full details of the stake pool, see [Staking](../concepts/staking.md)
```

Comment:
```suggestion
:::danger Delegation pools are permissionless and anyone can add stake. Stake pools cannot be changed once it's created, though it can be removed from the validator set and assets withdrawn. For full details of the stake pool, see [Staking](../concepts/staking.md)
```